nix/pkgs/core.nix

```diff
@@ -17,14 +17,14 @@ rec {
     export CARGO_BUILD_JOBS=4
 
     # ensure plain build works
-    cargo build --no-default-features --manifest-path=crates/holochain/Cargo.toml
+    # cargo build --no-default-features --manifest-path=crates/holochain/Cargo.toml
 
     # alas, we cannot specify --features in the virtual workspace
     cargo test warm_wasm_tests --manifest-path=crates/holochain/Cargo.toml --features slow_tests,build_wasms
```

Comment:
This is now included in the next invocation as the `warm_wasm_tests` merely acts as a filter, and the next invocation runs the identical feature set without any filter. I.e. this line can be removed now